test-suites/_base_interest_rate_strategy.spec.ts

```diff
@@ -238,6 +244,30 @@ describe("Interest Rate Tests", () => {
         )
       ).to.be.revertedWith(INVALID_OPTIMAL_USAGE_RATIO);
     });
+
+    it("TC-interest-rate-strategy-07 PoolConfigurator updates the ReserveInterestRateStrategy address", async () => {
+      const {pool, deployer, dai, configurator} = await loadFixture(
+        testEnvFixture
+      );
+
+      // Impersonate PoolConfigurator
+      await topUpNonPayableWithEther(
+        deployer.signer,
+        [configurator.address],
+        utils.parseEther("1")
+      );
+      await impersonateAccountsHardhat([configurator.address]);
+      const configSigner = await hre.ethers.getSigner(configurator.address);
+
+      expect(
+        await pool
+          .connect(configSigner)
+          .setReserveInterestRateStrategyAddress(dai.address, ZERO_ADDRESS)
+      );
```

Comment:
```suggestion
      expect(
        await pool
          .connect(configSigner)
          .setReserveInterestRateStrategyAddress(dai.address, ZERO_ADDRESS)
      ).to.be.eq(true);
```
`expect(x);`  inside It seems that no verification has been done。  Can we add some verifications

Reply:
well, that's the way Aave checked that simply a tnx succeeds, and maybe also kind of pointing the part under test, rather than leaving the test with no expect statement. As for `to.eq(true)`, I'm not convinced that's a better approach as would imply the function is returning a boolean, where is not. Maybe would be better, where possible, if the function emits some event, to use `.to.emit()`

Reply:
ok, As for to.eq(true) .Just an example